packages/portal/src/components/image/ImageEagerOrLazy.vue

```diff
@@ -0,0 +1,62 @@
+<template>
+  <component
+    :is="lazy ? 'b-img-lazy' : 'b-img'"
+    :alt="alt"
+    :blank-color="lazy ? blankColor : null"
+    :blank-width="lazy && width"
```

Comment:
these expressions end up throwing `Invalid prop: type check failed for prop "width". Expected Number, String, got Boolean with value false`

Reply:
Indeed. Commit [df5e46b](https://github.com/europeana/portal.js/pull/2326/commits/df5e46b2f9ac613bd7617446ae42b7023fbac66c) makes the values fall back to `null`